src/utils/eis_utils/src/eis_utils.c

```diff
@@ -468,208 +550,241 @@ EISUtilityResult RequestConnectionStringFromEISWithExpiry(
     JSON_Value* certResponseJson = NULL;
     char* certString = NULL;
 
-    EISErr identityResult = RequestIdentitiesFromEIS(timeoutMS, &identityResponseStr);
+    printf("Parent process (PID %d) is about to fork a child process...\n", getpid());
```

Comment:
In this function, consider using named-pipe for IPC between parent process and child process. I think the codes might be simpler.